libmbp/devices/pantech.cpp

```diff
@@ -39,6 +39,27 @@ namespace mbp
         device->setBootBlockDevs({ QCOM_BOOT, "/dev/block/mmcblk0p7" });
         device->setRecoveryBlockDevs({ QCOM_RECOVERY, "/dev/block/mmcblk0p14" });
         device->setExtraBlockDevs({ QCOM_TZ, "/dev/block/mmcblk0p8" });
+	device->twOptions()->supported = true;
```

Comment:
Please fix the indentation for these 10 lines so it uses spaces and matches the other lines :)